pdr_backend/pdr_dashboard/util/db.py

```diff
@@ -57,6 +58,11 @@ def __init__(self, ppss):
             addr for addr in ppss.predictoor_ss.my_addresses if addr in valid_addresses
         ]
 
+    def format_to_parquet_file_path(self, table_class: Type[LakeMapper]) -> str:
```

Comment:
please improve the DX...

1. rename this function into something simpler
self.tbl_parquet_path(Table)

2. perhaps move it to the TBL object...
```f"""SELECT * FROM {BronzePrediction.parquet_path}"""